query.go

```diff
@@ -48,7 +48,9 @@ func NewConjunctionQuery(conjuncts ...query.Query) *query.ConjunctionQuery {
 // NewDateRangeQuery creates a new Query for ranges
 // of date values.
 // Date strings are parsed using the DateTimeParser configured in the
-//  top-level config.QueryDateTimeParser
```

Comment:
ditto

Reply:
This is for the benefit of [gopls](https://pkg.go.dev/golang.org/x/tools/gopls)